components/content/Icon/Icon.stories.ts

```diff
@@ -0,0 +1,254 @@
+import { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+
+import './Icon';
+
+const description = `
+Wraps an svg icon to provide alignment and sizing.
+
+Icons must have the appropriate fill or stroke set to currentColor.
+
+Icons are hidden from screen readers by default and should be accompanied by text. If an icon must be used on its own, add an aria-label to the icon element.
+`;
+
+export default {
+  component: 'diamond-icon',
+  parameters: {
+    docs: {
+      description: {
+        component: description,
+      },
+    },
+  },
+  argTypes: {
+    shape: {
```

Comment:
Should this prop be changed to "variant" to match the API of other components?

Really it's not the shape of the icon it's adding a backdrop, circle is the only backdrop that seems to ever be used as convention, thinking about it a square could work in some situations but I never see it.